rdr_service/services/data_quality.py

```diff
@@ -195,7 +195,7 @@ def run_data_quality_checks(self, for_data_since: datetime = None):
                         f'but participant signed up at {participant_signup_time}'
                     )
                 if suspension_datetime and authored_time > suspension_datetime:
-                    logging.error(f'Response {response_id} authored for suspended participant')
+                    logging.error(f'Response {response_id} authored for suspended participant (module: {module_code})')
```

Comment:
If a withdrawal is a valid case, should there be a check that if the `module_code` is a withdrawal module (values `withdrawal_intro` or 'StopParticipating`) then there is no error logged?

Reply:
I think that would be a good next step, but for now I was just looking to get the logs to tell me what was responded to (so I didn't have an extra step of looking it up).